packages/presentation/src/components/Avatar.svelte

```diff
@@ -29,20 +31,30 @@
       url = blobURL
     })
   } else if (avatar !== undefined && avatar !== null) {
-    url = getFileUrl(avatar, size)
+    if (avatar.type === 'image') {
```

Comment:
You should use model and Provider based on schema, using `getResource(getProviderId(schema))`